calico/_includes/charts/calico/templates/calico-kube-controllers-rbac.yaml

```diff
@@ -71,6 +71,7 @@ rules:
       - ipreservations
     verbs:
       - list
+      - watch
```

Comment:
This change also needs to be added to the tigera/operator repo in `pkg/render/kubecontrollers/kube-controllers.go`

Reply:
Strictly speaking we don't need watch permissions on `ipreservations`, so we should split that out into a new section.

Similarly, the comment needs updating since we watch these resources for reasons other than just node deletion now.

Reply:
I've updated the permissions as per your feedback. Do you mind letting me know if the update that I made to the comment works for you? 

I see from the git history that the comment was put in place well before the current iteration of the controller, I tried to update it with my understanding of how things have changed but I could be off.

Reply:
yep I think the comments look alright to me.